qa/L1_FW_iterators_separate_executor/test.h

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -e
```

Comment:
Since the test is not strictly related to FW iterators I'd remove that from the test name. Maybe something like `L1_separate_executor`

Reply:
Done